moler/device/junipergeneric3.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+"""
+Juniper Generic module.
+"""
+
+__author__ = 'Sylwester Golonka, Jakub Kupiec, Marcin Usielski'
+__copyright__ = 'Copyright (C) 2019-2024, Nokia'
+__email__ = 'sylwester.golonka@nokia.com, jakub.kupiec@nokia.com, marcin.usielski@nokia.com'
+
+import logging
+from abc import ABCMeta
+from six import add_metaclass
```

Comment:
hmm, I'm just thinking if we still need `six` module if we don't support python2 anymore. <br>especially for new added files

Reply:
Removed. The comment is used instead of that code without any effect.

Reply:
Donr